src/Sylius/Bundle/ResourceBundle/Controller/ResourcesCollectionProvider.php

```diff
@@ -48,13 +49,20 @@ public function get(RequestConfiguration $requestConfiguration, RepositoryInterf
     {
         $resources = $this->resourcesResolver->getResources($requestConfiguration, $repository);
 
-        if ($resources instanceof Pagerfanta) {
+        if ($resources instanceof ResourceGridView or $resources instanceof Pagerfanta) {
```

Comment:
We cannot do that, the method need to return ResourceGridView.
I'll propose something in a minute.